pkg/apiclients/testapi/issues.go

```diff
@@ -732,6 +788,35 @@ func deduplicateStrings(slice []string) []string {
 	return result
 }
 
+// deduplicateDependencyPaths removes duplicate dependency paths from a slice.
+// Paths are considered duplicates if they have the same sequence of packages.
+func deduplicateDependencyPaths(paths [][]string) [][]string {
```

Comment:
I think this could be something like a generic deduplicateArray and live in a utility function.